src/core/react-query/file/mutations.ts

```diff
@@ -43,8 +43,8 @@ export const useDeleteFileLinkMutation = () =>
 
 export const useIgnoreFileMutation = () =>
   useMutation({
-    mutationFn: ({ fileId, ignore }: IgnoreFileRequestType) => axios.put(`File/${fileId}/Ignore`, { ignore }),
-    onSuccess: () => invalidateQueries(['files', { include_only: ['Ignored'] }]),
+    mutationFn: ({ fileId, ignore }: IgnoreFileRequestType) => axios.put(`File/${fileId}/Ignore?value=${ignore}`),
```

Comment:
```suggestion
    mutationFn: ({ fileId, ignore }: IgnoreFileRequestType) => axios.put(`File/${fileId}/Ignore`, { value: ignore }),
```

Reply:
You are suggesting to revert it to the way it didn't work. It needs a get parameter it doesn't work otherwise. Swagger says that value is located in path i.e. in url not a post parameter.

Reply:
```suggestion
    mutationFn: ({ fileId, ignore }: IgnoreFileRequestType) =>
      axios.put(`File/${fileId}/Ignore`, undefined, { params: { value: ignore } }),
```

my bad, it should actually be this. I thought it was a `get` and not `put`